plugin/othttp/handler.go

```diff
@@ -50,15 +50,15 @@ type Filter func(*http.Request) bool
 // the mux are wrapped with WithRouteTag. A Handler will add various attributes
 // to the span using the core.Keys defined in this package.
 type Handler struct {
-	operation string
-	handler   http.Handler
+	handler http.Handler
 
 	tracer           trace.Tracer
 	props            propagation.Propagators
 	spanStartOptions []trace.StartOption
 	readEvent        bool
 	writeEvent       bool
 	filters          []Filter
+	spanFormatter    func(*http.Request) string
```

Comment:
I think that `Handler` should preserve the `operation` member and the span formatter should receive the operation name string as a parameter too, otherwise the `operation` parameter in the `NewHandler` function becomes useless if we pass a custom span formatter.

Also, shouldn't it be called `spanNameFormatter`?

Reply:
@krnowak the way I thought about it is that if a user passes "othttp.WithSpanFormatter", then they can assume "operation" is overwritten, but I will change that into your suggestion. 

Also, `spanNameFormatter` sounds good, should we also change the option name from `othttp.WithSpanFormatter` into `othttp.WithSpanNameFormatter`?

Reply:
> Also, `spanNameFormatter` sounds good, should we also change the option name from `othttp.WithSpanFormatter` into `othttp.WithSpanNameFormatter`?

Yes, please.